source/audio/__init__.py

```diff
@@ -8,9 +8,47 @@
 	_setSoundSplitState,
 	_toggleSoundSplitState,
 )
+from . import appsVolume, soundSplit, utils
+import atexit
+import nvwave
+from pycaw.utils import AudioUtilities
+from comtypes import COMError
+from logHandler import log
 
 __all__ = [
 	"SoundSplitState",
 	"_setSoundSplitState",
 	"_toggleSoundSplitState",
 ]
+
+audioUtilitiesInitialized: bool = False
+
+
+def initialize() -> None:
+	if nvwave.usingWasapiWavePlayer():
+		try:
+			AudioUtilities.GetAudioSessionManager()
+		except COMError:
+			log.exception("Could not initialize audio session manager")
+			return
+		log.debug("Initializing utils")
+		utils.initialize()
+		log.debug("Initializing appsVolume")
+		appsVolume.initialize()
+		log.debug("Initializing soundSplit")
+		soundSplit.initialize()
+		global audioUtilitiesInitialized
+		audioUtilitiesInitialized = True
+	else:
+		log.debug("Cannot initialize audio utilities as WASAPI is disabled")
+

```

Comment:
To fix lint.
- Go to appVeyor details of this PR
- click "Artifacts"
- Search for "Flake8.txt" and click it to open it.

The only item to fix is to add a second blank line here:

```suggestion


```